### PR TITLE
Freeze to 0.8.7 ver

### PR DESCRIPTION
**Pull request by shubham3121**
## Description
Please include a summary of the change, the motivation, and any additional context that will help others understand your PR. If it closes one or more open issues, [please tag them as described here](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword).

## Affected Dependencies
List any dependencies that are required for this change.

## How has this been tested?
- Describe the tests that you ran to verify your changes.
- Provide instructions so we can reproduce.
- List any relevant details for your test configuration.

## Checklist
- [ ] I have followed the [Contribution Guidelines](https://github.com/OpenMined/.github/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/OpenMined/.github/blob/master/CODE_OF_CONDUCT.md)
- [ ] I have commented my code following the [OpenMined Styleguide](https://github.com/OpenMined/.github/blob/master/STYLEGUIDE.md)
- [ ] I have labeled this PR with the relevant [Type labels](https://github.com/OpenMined/.github/labels?q=Type%3A)
- [ ] My changes are covered by tests

### CODE REVIEW

File: .github/workflows/pr-tests-stack.yml [hunk 1]
```diff
@@ -56,11 +56,11 @@ jobs:
           pip install uv==0.2.17 tox tox-uv==1.9.0
           uv --version
 
-      - name: Run syft backend base image building test
```
Comment (rasswanth-s) [inline .github/workflows/pr-tests-stack.yml:59]: We might also need to disable the pool_image_test.py

Comment (rasswanth-s) [reply .github/workflows/pr-tests-stack.yml:59]: which contains 0.8.6 tags

Comment (shubham3121) [reply .github/workflows/pr-tests-stack.yml:59]: Rasswanth, I think we skip it bcz we run `pool_image_test.py` with k8s only(not with docker anymore), hence the tag is selected from the k3d registry.

https://github.com/OpenMined/PySyft/blob/4293519dca308b676d221593e0431ec9f482196c/tests/integration/container_workload/pool_image_test.py#L27

**Review by rasswanth-s — APPROVED** [review]
🚀

**Pull request merged**